app/models/api/tta.rb

```diff
@@ -1,21 +1,52 @@
 require "fake_api_response_wrapper"
 
-class Api::Tta
-  def initalize(**)
+module Api::Tta
+  def host
+    Rails.configuration.x.tta.api_hostname
   end
 
-  class ActivityReport
-    include FakeApiResponseWrapper
-    attr_accessor :display_id
+  def port
+    Rails.configuration.x.tta.api_port
+  end
 
-    def initialize(display_id:)
-      @display_id = display_id
-    end
+  def use_ssl?
+    Rails.configuration.x.tta.use_ssl
+  end
+
+  def configure_auth(request)
+    request["Authorization"] = "Bearer #{access_token.bearer_token}"
+  end
+end
+
+class Api::Tta::ActivityReport < ApiRequest
+  include Api::Tta
+  attr_accessor :display_id, :access_token
 
-    def request
-      details_wrapper.merge(
-        data: Api::FakeData::ActivityReport.new(display_id: display_id).data
-      )
+  def initialize(display_id:, access_token:)
+    @display_id = display_id
+    @access_token = access_token
+  end
+
+  def request
+    Rails.logger.debug <<~EODM
+      TTAHub #{path}:
+      #{JSON.pretty_generate(response)}
+    EODM
+    if response[:code] == "200"
+      response[:body]
+    else
+      Rails.logger.error "TTA Hub call to #{path} responded with #{response[:code]}"
+      fail response[:body].to_json
```

Comment:
You are correct that they are aliases.

My personal rule of thumb (that I likely violate in other places) is that `fail` means "This is where a failure originated from" and `raise` means, "I rescued this error and did something with it, but I want to continue it on". Since they're aliases though that distinction is lost on the code and is only for humans